packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -230,7 +226,12 @@ export class PluginLedgerConnectorQuorum
     req: InvokeContractV1Request,
   ): Promise<any> {
     const fnTag = `${this.className}#invokeContract()`;
-    const contractName = req.contractName;
+    if (req.contractName == undefined) {
```

Comment:
@elenaizaguirre Check for  falsy-ness with the exclamation mark, it's more robust

Reply:
Done!